packages/@aws-cdk/aws-cloudwatch/lib/alarm.ts

```diff
@@ -9,10 +9,22 @@ import { dropUndefined } from './private/object';
 import { MetricSet } from './private/rendering';
 import { parseStatistic } from './private/statistic';
 
+/**
+ * Interface for Alarm Rule.
+ */
+export interface IAlarmRule {
+
+  /**
+   * serialized representation of Alarm Rule to be used when building the Composite Alarm resource.
+   */
+  toAlarmRule(): string;
```

Comment:
Since we also have an `AlarmRule` object, I would half-and-half expect `toAlarmRule()` to *return* an `AlarmRule` object, but it doesn't. It returns a string.

Can we call this `renderAlarmRule()` instead?